### PR TITLE
Added missing project "Labyrinth"

### PR DESCRIPTION
**Pull request by yashk0thari**

### CODE REVIEW

**Comment by yashk0thari** [comment]
https://beta-king.github.io/Finale.fossasia.org/ --> Preview Link !!

**Comment by yashk0thari** [comment]
please review :>

**Comment by yashk0thari** [comment]
Please review :D

File: index.html [hunk 1]
```diff
@@ -95,6 +95,7 @@
 											<li><a target="_self" href="https://query-server.herokuapp.com/">Query Server</a></li>
 											<li><a target="_self" href="http://knitting.fossasia.org/">Knitting Lab</a></li>
 											<li><a target="_self" href="https://badgeyay-dev.herokuapp.com/">Badgeyay</a></li>
+							     				<li><a target="_self" href="https://github.com/fossasia/labyrinth">Labyrinth</a></li>
```
Comment (abishekvashok) [inline index.html:98]: Pls wait till https://github.com/fossasia/labyrinth/issues/38 is done so that you can add the deployment link

Comment (yashk0thari) [reply index.html:98]: ohk ;D

**Review by sayak119 — COMMENTED** [review]
Kindly mention the issue number in your PR

**Pull request closed without merging**